src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrBackflower.cs

```diff
@@ -112,7 +112,7 @@ public async Task<bool> FlowBackAsync(
 
         Codeflow lastFlow = await GetLastFlowAsync(mapping, targetRepo, currentIsBackflow: true);
 
-        return await FlowBackAsync(
+        return (await FlowBackAsync(
```

Comment:
Can we return the `CodeFlowResult` here too? And only use the boolean from within wherever we consume this API